### DISCUSSION TITLE
openlara

### DISCUSSION THREAD

**Discussion by glyner** [eduardofilo/RG350_adam_image#107] [Ports]
openlara fails to start - worked on stock firmware so not sure if this an issue with od beta?

**Comment by eduardofilo** [comment]
OD beta was not supported in OpenLara. But @plrguez has managed to compile it. He will publish the changes to code in a release in a day or two.

**Comment by glyner** [comment]
awesome, thanks for the update

**Comment by plrguez** [comment]
I have made a release, see [here](https://github.com/plrguez/OpenLara/releases/tag/20211103-od-v1.0.0)
Choose OpenGL version for best performance. Shadows option have great impact on performance so best leave it with "low" value.
